api/app/models/env_canada.py

```diff
@@ -227,7 +226,8 @@ def check_if_model_run_complete(self, urls):
         # pylint: disable=no-member
         actual_count = get_processed_file_count(self.session, urls)
         expected_count = len(urls)
-        logger.info('we have processed %s/%s files', actual_count, expected_count)
+        logger.info('we have processed %s/%s files',
```

Comment:
What are you using for autoformatting? I think it's cutting things shorter than they need to be.
```
"python.formatting.autopep8Args": [
        "--max-line-length=110"
    ]
```